src/main/java/net/earthcomputer/clientcommands/command/CTimeCommand.java

```diff
@@ -1,13 +1,19 @@
 package net.earthcomputer.clientcommands.command;
 
 import com.mojang.brigadier.CommandDispatcher;
+import com.mojang.brigadier.arguments.IntegerArgumentType;
+
+import net.earthcomputer.clientcommands.features.ClientTimeModifier;
 import net.fabricmc.fabric.api.client.command.v1.FabricClientCommandSource;
 import net.minecraft.SharedConstants;
 import net.minecraft.client.MinecraftClient;
+import net.minecraft.command.argument.TimeArgumentType;
```

Comment:
This import is unused since the `TimeArgumentType.time` method is imported statically instead.